code-ready/crc
# Backup log file as part of delete command

crc log file is opened as append mode and never gets deleted until
user manually delete it or remove the base directory. Currently it
is filled with lot of message which might not be require to debug
an issue. This patch make sure every new crc cycle (setup => start => delete)
have required logs to logfile.

## Testing
- Run crc start => check the log file exist in `~/. crc/crc.log`
- Run crc delete => check the log file shouldn't present

Any other command shouldn't delete this file.
